tasks/run_cd4pe_job.rb

```diff
@@ -200,9 +201,27 @@ def initialize(working_dir:, job_token:, web_ui_endpoint:, job_owner:, job_insta
         file.write(ca_cert)
       end
     end
+
+    set_home_env_var
+    set_repo_dir_env_var
     
   end
 
+  def set_home_env_var
+    # when the puppet orchestrator runs a Bolt task, it does so as a user without $HOME set.
+    # We need to ensure $HOME is set so jobs that rely on this env var can succeed.
+    if (@windows_job)
+      # if windows, we can rely on powershell
+      ENV['HOME'] = run_system_cmd("powershell (Resolve-Path ~")[:message]
```

Comment:
@chrisleicester Man, we forgot a `)` here 😆 

We have `(Resolve-Path ~`

And we need `(Resolve-Path ~)`

Reply:
Also why does this silently fail

Reply:
addressed in https://github.com/puppetlabs/puppetlabs-cd4pe_jobs/pull/14/